src/TeamSpeak3QueryApi/Specialized/Responses/GetClientDetailedInfo.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TeamSpeak3QueryApi.Net.Specialized.Responses
+{
+    public class GetClientDetailedInfo : Response
+    {
+        [QuerySerialize("cid")]
+        public int ChannelId;
+
+        [QuerySerialize("client_unique_identifier")]
+        public string UniqueIdentifier;
+
+        [QuerySerialize("client_nickname")]
+        public string NickName;
+
+        [QuerySerialize("client_login_name")]
+        public string ServerQueryName;
+
+        [QuerySerialize("client_database_id")]
+        public int DatabaseId;
+
+        [QuerySerialize("client_type")]
+        public ClientType Type;
+
+        [QuerySerialize("client_version")]
+        public string Version;
+
+        [QuerySerialize("connection_client_ip")]
+        public string ConnectionIp;
+
+        [QuerySerialize("client_platform")]
+        public string Plattform;
+
+        [QuerySerialize("client_description")]
+        public string Description;
+
+        [QuerySerialize("client_input_muted")]
+        public bool InputMuted;
+
+        [QuerySerialize("client_output_muted")]
+        public bool OutputMuted;
+
+        [QuerySerialize("client_outputonly_muted")]
+        public bool OutputOnlyMuted;
+
+        [QuerySerialize("client_is_recording")]
+        public bool IsRecording;
+
+        [QuerySerialize("client_servergroups")]
+        private string _serverGroupsStr;
+        public List<int> ServerGroupIds
+        {
+            get {
+                if(string.IsNullOrWhiteSpace(_serverGroupsStr))
+                    return new List<int>();
+
+                return _serverGroupsStr.Split(',').Select(grp => int.Parse(grp)).ToList();
+            }
+            private set { }
+        }
+
+        [QuerySerialize("client_channel_group_id")]
+        private string _channelGroupsStr; 
+        public List<int> ChannelGroupsIds
+        {
+            get
+            {
+                if (string.IsNullOrWhiteSpace(_channelGroupsStr))
+                    return new List<int>();
+
+                return _channelGroupsStr.Split(',').Select(grp => int.Parse(grp)).ToList();
+            }
```

Comment:
Is it really needed to do the entire computation in the getter? How about adding a new converter that converts to `IReadOnlyList<int>`? Then, declaring the property is all we need.